Documentation/specs/runtime-configuration-file.md

```diff
@@ -25,8 +25,13 @@ The files are both JSON files stored in UTF-8 encoding. Below are sample files.
 ```json
 {
     "runtimeOptions": {
-        "gcServer": true,
-        "gcConcurrent": false,
+
+        "System.GC.Server": true,
+        "System.GC.Concurrent": false,
+        "System.Threading.ThreadPool.MinThreads": 4,
+        "System.Threading.ThreadPool.MaxThreads": 8,
+        "System.Threading.Thread.UseAllCpuGroups": true
+
         "framework": {
             "name": "Microsoft.DotNetCore",
```

Comment:
FYI the `runtimeOptions` section is also defined in `project.json` and the format is expected to match exactly. It is covered in some of the specs @DamianEdwards wrote, so if we're changing stuff **outside** of that section we should make sure to consider that. Within the `runtimeOptions` section, we can :tada: :arrow_down: